### PR TITLE
Ai can ping

### LINKED ISSUE #18180: AI can't emote
#### Description of issue
AIs can't emote at all. They don't even have a `*help` thingy.
They should have at least as many emotes as pAIs have.

#### Issue bingo

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by PublicEnemyMaiPai**
closes #18180
lets AI Buzz Ping Beep 

:cl:
 * rscadd: AI can now Beep Ping Buzz

### CODE REVIEW

**Comment by PublicEnemyMaiPai** [comment]
plz tell me i didnt fuck it up again lol

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+				param = null
+
+			if (param)
+				message = "<B>[src]</B> buzzes at [param]."
+			else
+				message = "<B>[src]</B> buzzes."
+			playsound(get_turf(src), 'sound/machines/buzz-sigh.ogg', 50, 0)
+			m_type = VISIBLE
+
+		if("law")
```
Comment (jknpj) [inline code/modules/mob/living/silicon/ai/emote.dm]: You're not the law, pal.

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+			message = "<B>[src]</B> shows its legal authorization barcode."
+
+			playsound(get_turf(src), 'sound/voice/biamthelaw.ogg', 50, 0)
+			m_type = VISIBLE
+
+		if("halt")
```
Comment (jknpj) [inline code/modules/mob/living/silicon/ai/emote.dm]: ditto

**Comment by PublicEnemyMaiPai** [comment]
well i couldnt have it check for robocop, otherwise its checking for being a secborg, and who am I to deny the malf its right to tell its attackers it is the law and they should halt

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if (param)
+				message = "<B>[src]</B> beeps at [param]."
+			else
+				message = "<B>[src]</B> beeps."
+			playsound(get_turf(src), 'sound/machines/twobeep.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: Change `get_turf(src)` -> `src`

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if (param)
+				message = "<B>[src]</B> pings at [param]."
+			else
+				message = "<B>[src]</B> pings."
+			playsound(get_turf(src), 'sound/machines/ping.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: Change `get_turf(src)` -> `src`

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if (param)
+				message = "<B>[src]</B> buzzes at [param]."
+			else
+				message = "<B>[src]</B> buzzes."
+			playsound(get_turf(src), 'sound/machines/buzz-sigh.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: Change `get_turf(src)` -> `src`

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+			m_type = VISIBLE
+
+		if("law")
+			message = "<B>[src]</B> shows its legal authorization barcode."
+
+			playsound(get_turf(src), 'sound/voice/biamthelaw.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: Change `get_turf(src)` -> `src`

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+			m_type = VISIBLE
+
+		if("halt")
+			message = "<B>[src]</B>'s speakers skreech, \"Halt! Security!\"."
+
+			playsound(get_turf(src), 'sound/voice/halt.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: Change `get_turf(src)` -> `src`

**Review by DamianX — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if (param)
+				message = "<B>[src]</B> buzzes at [param]."
+			else
+				message = "<B>[src]</B> buzzes."
+			playsound(src), 'sound/machines/buzz-sigh.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: remove the first `)` after `src`

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if (param)
+				message = "<B>[src]</B> pings at [param]."
+			else
+				message = "<B>[src]</B> pings."
+			playsound(src), 'sound/machines/ping.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: remove the first `)` after `src`

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if (param)
+				message = "<B>[src]</B> beeps at [param]."
+			else
+				message = "<B>[src]</B> beeps."
+			playsound(src), 'sound/machines/twobeep.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: remove the first `)` after `src`

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+		if("law")
+			message = "<B>[src]</B> shows its legal authorization barcode."
+
+			playsound(src), 'sound/voice/biamthelaw.ogg', 50, 0)
+			m_type = VISIBLE
+
+		if("halt")
+			message = "<B>[src]</B>'s speakers skreech, \"Halt! Security!\"."
+
+			playsound(src), 'sound/voice/halt.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: remove the first `)` after `src`

File: code/modules/mob/living/silicon/ai/emote.dm [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+				message = "<B>[src]</B> buzzes at [param]."
+			else
+				message = "<B>[src]</B> buzzes."
+			playsound(src), 'sound/machines/buzz-sigh.ogg', 50, 0)
+			m_type = VISIBLE
+
+		if("law")
+			message = "<B>[src]</B> shows its legal authorization barcode."
+
+			playsound(src), 'sound/voice/biamthelaw.ogg', 50, 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/emote.dm]: remove the first `)` after `src`

**Review by DamianX — CHANGES_REQUESTED** [review]

**Comment by DamianX** [comment]
I second jk's opinion: `law` and `halt` seem out of place.

**Comment by PublicEnemyMaiPai** [comment]
I suppose i can make the correction requested when i get home then.

**Review by DamianX — APPROVED** [review]

**Pull request merged**